.env.example

```diff
@@ -25,3 +25,8 @@ AUTH_SECRET=
 # https://turbo.build/repo/docs/core-concepts/remote-caching#artifact-integrity-and-authenticity-verification
 # This can also be generated with `openssl rand -hex 32`, but do not re-use the value from AUTH_SECRET
 TURBO_REMOTE_CACHE_SIGNATURE_KEY=
+
+# Controls the default behavior of Link components when a `prefetch` prop is not provided.
+# If set to `true`, Link components will default to `prefetch=true`.
+# Otherwise, Link components will default to `prefetch=false`.
+DEFAULT_PREFETCH_BEHAVIOR=false
```

Comment:
We need to prefix this with `NEXT_PUBLIC_`

Reply:
🤦